htdocs_symfony/src/Components/Subscriber/MenuSubscriber.php

```diff
@@ -19,100 +20,113 @@ class MenuSubscriber implements EventSubscriberInterface
      */
     private $security;
 
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
     /**
      * MenuSubscriber constructor.
      *
      * @param Security $security
+     * @param TranslatorInterface $translator
      */
-    public function __construct(Security $security)
+    public function __construct(Security $security, TranslatorInterface $translator)
     {
         $this->security = $security;
+        $this->translator = $translator;
     }
 
     /**
      * @return array[]
      */
-    public static function getSubscribedEvents(): array
+    public static function getSubscribedEvents()
+    : array
     {
         return [
             KnpMenuEvent::class => ['onSetupMenu', 100],
         ];
     }
 
+    /**
+     */
+    private function addMenuItem()
+    {
+    }
+
     /**
      * @param KnpMenuEvent $event
      */
     public function onSetupMenu(KnpMenuEvent $event)
     {
         $menu = $event->getMenu();
 
-        $menu->addChild('MainNavigationMenuItem', [
-            'label' => 'MAIN NAVIGATION',
-            'childOptions' => $event->getChildOptions()
-        ])->setAttribute('class', 'header');
+        $menu->addChild('cacheZeug', [
+            'label' => 'Cachezeugs',
+            'route' => 'backend_caches_index',
+            'childOptions' => $event->getChildOptions(),
+        ])->setLabelAttribute('icon', 'fas fa-map-marker-alt');
 
-        if ($this->security->isGranted("CAN_VIEW", UserEntity::class)) {
```

Comment:
Warum fliegt das raus?

Reply:
Da den Menüeintrag jeder sehen darf, hab ich die Abfrage rausgeschmissen. Andere Menüeinträge sind beschränkt zugänglich und wurden mit "$this->security->isGranted('ROLE_*')" umschlossen.